packages/devtools_app/lib/src/memory/flutter/memory_graph_model.dart

```diff
@@ -0,0 +1,439 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:vm_service/vm_service.dart';
+
+import '../flutter/memory_controller.dart';
+
+// TODO(terry): Ask Ben, what is a class name of ::?
+/// Internal class names :: automatically filter out.
+const String internalClassName = '::';
+
+// TODO(terry): Bake in types instead of comparing a fully qualified class name.
+const String predefinedNull = 'dart:core,Null';
+const String predefinedString = 'dart:core,_OneByteString';
+const String predefinedList = 'dart:core,_List';
+const String predefinedMap = 'dart:collection,_InternalLinkedHashMap';
+
+class Predefined {
+  const Predefined(this.prettyName, this.isScalar);
+
+  final String prettyName;
+  final bool isScalar;
+}
+
+/// Structure key is fully qualified class name and the value is
+/// a List first entry is pretty name known to users second entry
+/// is if the type is a scalar.
+const Map<String, Predefined> predefinedClasses = {
+  'dart:core,bool': Predefined('bool', true),
+  // TODO(terry): Handle Smi too (Integer)?
+  // Integers not Smi but fit into 64bits.
+  'dart:core,_Mint': Predefined('int', true),
+  'dart:core,_Double': Predefined('Double', true),
+  predefinedString: Predefined('String', true),
+  predefinedList: Predefined('List', false),
+  predefinedMap: Predefined('Map', false),
+};
+
+/// List of classes to monitor, helps to debug particular class structure.
+final Map<int, String> _monitorClasses = {};
+
+/// Ensure the classId is zero based ()
+bool monitorClass({int classId, String className, String message = ''}) {
+  if (classId != null) {
+    if (_monitorClasses.containsKey(classId)) {
+      final className = _monitorClasses[classId];
+      print('STOP: class $className [classId=$classId]  $message');
```

Comment:
remove print statements here and elsewhere before committing

Reply:
This particular print I'd like to keep its for monitoring classes.  I'll add a test check for this output it means the user has forgot to turn off monitoring - a debug only thing.